modules/server/src/main/scala/navigate/server/tcs/TcsEpicsSystem.scala

```diff
@@ -1044,6 +1192,47 @@ object TcsEpicsSystem {
       channels.oiProbeTracking
     )
     override val oiwfsProbeCmds: ProbeCommandsChannels[F] = buildProbeCommandsChannels(channels.telltale, channels.oiProbe)
+
+    override val m1GuideConfigCmd: Command4Channels[F, String, String, Int, String] = Command4Channels(
+      channels.telltale,
```

Comment:
I guess this is the "witness" to tell if the system is connected?

Reply:
Yes, that's the one.